packages/survey/src/Question/ItemGroup/models/ItemGroupProps.ts

```diff
@@ -0,0 +1,14 @@
+import { UseCheckboxGroupProps, UseRadioGroupProps } from '@chakra-ui/react'
+
+import { ItemProps, OptionObj } from '../../Item'
+
+export type ItemGroupProps = {
+  /* An array with all available options for the group component */
+  options: OptionObj[]
+  /* The direction how the group component should be rendered */
+  direction?: 'vertical' | 'horizontal'
+  /* The max columns to set (when images are used) */
```

Comment:
JSDoc like described above

```suggestion
  /** An array with all available options for the group component. */
  options: OptionObj[]
  /** The direction how the group component should be rendered. */
  direction?: 'vertical' | 'horizontal'
  /** The max columns to set (when images are used). */
```

Reply:
done!